src/v2/Components/RelatedCollectionsRail/RelatedCollectionsRail.tsx

```diff
@@ -38,14 +37,15 @@ export const RelatedCollectionsRail: React.FC<RelatedCollectionsRailProps> = pro
 
   if (collectionsWithArtworks.length > 3) {
     return (
-      <Box>
+      <>
+        {/* FIXME: Must remove this dependency */}
         <Waypoint onEnter={once(trackImpression)} />
```

Comment:
Assuming this is what's actually crashing the page. We should probably just add an intersection observer hook at some point.